core/dbt/task/run_operation.py

```diff
@@ -0,0 +1,40 @@
+from dbt.logger import GLOBAL_LOGGER as logger
+
+from dbt.task.base_task import BaseTask
+from dbt.adapters.factory import get_adapter
+from dbt.loader import GraphLoader
+
+import dbt
+import dbt.utils
+import dbt.exceptions
+
+
+class RunOperationTask(BaseTask):
+    def _get_macro(self):
```

Comment:
this fn is badly named. i would expect `_get_macro` to return a macro. this just processes the macro path into something usable